src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/response/RcaResponseTest.java

```diff
@@ -0,0 +1,109 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.response;
```

Comment:
please add the Amazon copyright header to any newly created test file

Reply:
Done